src/AspNetCore/Src/applicationmanager.cxx

```diff
@@ -171,6 +173,19 @@ APPLICATION_MANAGER::GetApplicationInfo(
                     strEventMsg.QueryStr());
             }
         }
+        else if (m_fInShutdown)
+        {
+            if (SUCCEEDED(strEventMsg.SafeSnwprintf(
```

Comment:
how could this in shutdown happen?
you may want to create a different error message for such situation

Reply:
This shutdown can happen through a global event. We don't want to call shutdown multiple times if this happens.